src/accelerate/big_modeling.py

```diff
@@ -302,6 +302,7 @@ def dispatch_model(
     offload_dir: Optional[Union[str, os.PathLike]] = None,
     offload_index: Optional[Dict[str, str]] = None,
     offload_buffers: bool = False,
+    cpu_offload: bool = False,
```

Comment:
I think it can be confusing as CPU offloading is already indicated in the `device_map`.

IMO ideally there should not be any argument added, and by default the weights of modules offloaded on RAM should be on `cpu` device, not `meta`. it this is kind of a breaking change in case anybody is assuming that by default attached weights are on `meta` and `weights_map` holds the true weights.